ui/desktop/electron-app/yarn.lock

```diff
@@ -3269,7 +3269,7 @@ tar-stream@^1.5.2:
     to-buffer "^1.1.1"
     xtend "^4.0.0"
 
-tar@^6.0.5, tar@^6.1.11, tar@^6.1.2, tar@^6.1.9:
```

Comment:
@lisbet-alvarez interesting we are seeing a dependency change here when your changes does not add anything related 🤔

Reply:
I think the main is out of sync, I also got this update locally

Reply:
Yup, change happened when I used `yarn` in order to use `yarn test`